packages/gatsby-plugin-sharp/src/image-data.ts

```diff
@@ -250,6 +251,8 @@ export async function generateImageData({
       args: {
         ...options,
         ...args.transformOptions,
```

Comment:
Why are remove `args` in the change above but not here? Seems inconsistent, wonder if there's a reason for it.